syntax = "proto3";

option go_package = "google.golang.org/grpc/examples/helloworld/helloworld";
option java_multiple_files = true;
option java_package = "io.grpc.examples.helloworld";
option java_outer_classname = "HelloWorldProto";

service Payment {
  rpc PagarTodoCarro (Nones) returns (Respuesta) {}
  rpc PagarItemEnElCarro(ItemId) returns (Respuesta) {}
}

message ItemId {
  int32 pedidoId =1;
}
message Nones{}

message Respuesta{
  int32 status_code = 1;
  string message =2;
}